### ISSUE TITLE
Unable to build

### ISSUE THREAD

**Issue by seblm** [xebia-france/xwhois#6]
Unable to build the project. Maybe I'm doing it wrong. Here is what I get on a fresh cloned repo:

```
$ npm install gulp
gulp@3.8.11 node_modules/gulp
├── v8flags@2.0.2
├── pretty-hrtime@0.2.2
├── interpret@0.3.10
├── deprecated@0.0.1
├── archy@1.0.0
├── minimist@1.1.0
├── tildify@1.0.0 (user-home@1.1.1)
├── semver@4.3.0
├── chalk@0.5.1 (escape-string-regexp@1.0.2, ansi-styles@1.1.0, supports-color@0.2.0, has-ansi@0.1.0, strip-ansi@0.3.0)
├── orchestrator@0.3.7 (stream-consume@0.1.0, sequencify@0.0.7, end-of-stream@0.1.5)
├── liftoff@2.0.1 (extend@1.3.0, flagged-respawn@0.3.1, resolve@1.0.0, findup-sync@0.2.1)
├── vinyl-fs@0.3.13 (graceful-fs@3.0.5, strip-bom@1.0.0, mkdirp@0.5.0, vinyl@0.4.6, defaults@1.0.0, glob-stream@3.1.18, through2@0.6.3, glob-watcher@0.0.6)
└── gulp-util@3.0.3 (array-differ@1.0.0, object-assign@2.0.0, beeper@1.0.0, array-uniq@1.0.2, replace-ext@0.0.1, lodash.reescape@3.0.1, lodash.reevaluate@3.0.1, lodash.reinterpolate@3.0.1, vinyl@0.4.6, dateformat@1.0.11, through2@0.6.3, multipipe@0.1.2, lodash.template@3.2.0)
$ gulp
[15:56:13] Failed to load external module coffee-script/register

/Users/seb/src/xebia-france/xwhois/gulpfile.coffee:4
# use dependencies
^
SyntaxError: Unexpected token ILLEGAL
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Liftoff.handleArguments (/usr/local/lib/node_modules/gulp/bin/gulp.js:116:3)
    at Liftoff.<anonymous> (/usr/local/lib/node_modules/gulp/node_modules/liftoff/index.js:179:16)
    at module.exports (/usr/local/lib/node_modules/gulp/node_modules/liftoff/node_modules/flagged-respawn/index.js:17:3)
    at Liftoff.<anonymous> (/usr/local/lib/node_modules/gulp/node_modules/liftoff/index.js:172:9)
```

**Comment by mbreton** [comment]
It seems that the coffee-script dependency is missing. Try to run the command "npm install coffee-script --save-dev" before and re-run the gulp command.
And don't forget to commit/push the modifications in the "package.json" made by the first command.

**Comment by seblm** [comment]
Trying to get rid of coffee doesn't helps:

```
$ coffee --compile gulpfile.coffee
$ gulp

Error: Cannot find module 'gulp-load-plugins'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/seb/src/xebia-france/xwhois/gulpfile.js:8:7)
    at Object.<anonymous> (/Users/seb/src/xebia-france/xwhois/gulpfile.js:213:4)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
```

```
$ head -n 8 gulpfile.js
// Generated by CoffeeScript 1.9.1
(function() {
  'use strict';
  var $, args, bowerFiles, browserSync, css, from, gulp, karma, log, ng, paths, reload, rm, sequence, task, to, watch;

  gulp = require('gulp');

  $ = require('gulp-load-plugins')();
```

**Comment by seblm** [comment]
Thanks @mbreton: instead of `[15:56:13] Failed to load external module coffee-script/register` now I get `[16:07:37] Requiring external module coffee-script/register`.
Issue is still `Error: Cannot find module 'gulp-load-plugins'`

**Comment by admin-codeship-xebia** [comment]
arf ...

npm install --save-dev gulp-load-plugins gulp-if-else del run-sequence
karma gulp-minify-css gulp-ng-annotate gulp-ng-templates gulp-ng-config
main-bower-files browser-sync

Mathieu Breton

[image: Xebia logo] http://www.xebia.fr/
« Software Development Done Right »
156, boulevard Haussmann, 75008 Paris
+33(0)6 50 17 49 31 / mbreton@xebia.fr youremail@xebia.fr
[image: Alliance logo]

**Comment by mbreton** [comment]
ERRATUM : you just need to execute a "npm install" command

Mathieu Breton

[image: Xebia logo] http://www.xebia.fr/
« Software Development Done Right »
156, boulevard Haussmann, 75008 Paris
+33(0)6 50 17 49 31 / mbreton@xebia.fr youremail@xebia.fr
[image: Alliance logo]

**Comment by seblm** [comment]
`npm install` works fine. Thanks @mbreton.
Here is next error :smiley: 

```
$ gulp
[18:12:14] Requiring external module coffee-script/register
[18:12:16] Build in development mode.
[18:12:16] Debug is disabled.
[18:12:16] Paths are :
 { bower: 'src/vendors/bower_components',
  web: 'src/app',
  conf: 'src/conf',
  build: 'build',
  exploded: 'build/exploded',
  test: 'test' }
[18:12:16] Using gulpfile ~/xwhois/gulpfile.coffee
[18:12:16] Starting 'build'...
[18:12:16] Starting 'clean:before-build'...
[18:12:16] Finished 'clean:before-build' after 2.64 ms
[18:12:16] Starting 'compile:sass'...

stream.js:94
      throw er; // Unhandled stream error in pipe.
            ^
Error: file to import not found or unreadable: bourbon
Current dir: ~/xwhois/src/app/styles/
```

**Comment by seblm** [comment]
For a unknown reason bourbon have change his path. Once fixed by 80b380d, it passes on my machine.

**Comment by alexandrehebert** [comment]
Je suis en train de mettre tout ça d'équerre et nettoyer les dépendances.
J'avais des dépendances en cache qui me jouent de sales tours.
